portfolio/src/main/webapp/script.js

```diff
@@ -12,21 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+async function getFormMessage() {
+  const response = await fetch('/data');
+  const quote = await response.text();
+  console.log(quote)
```

Comment:
2 things:

1. does your linter not catch lines that don't have a terminating `;`?
2. you should remove console logs from merged code

Reply:
No it doesn't, and they run without errors, super weird
Yes I'll remove console logs